pkg/virtualization/network/virtualnetworkadapter/syntheticnetworkadapter.go

```diff
@@ -46,3 +47,12 @@ func (sna *SyntheticNetworkAdapter) SetMACAddress(mac string) (err error) {
 func (sna *SyntheticNetworkAdapter) Rename(newName string) (err error) {
 	return sna.SetPropertyElementName(newName)
 }
+
+func (sna *SyntheticNetworkAdapter) GetGuestNetworkAdapterConfiguration() (guestConfiguration *GuestNetworkAdapterConfiguration, err error) {
+	wmiGuestConfig, err := sna.GetRelated("Msvm_GuestNetworkAdapterConfiguration")
```

Comment:
Please follow the pattern of closing these wmi instances. WMI code change is very sensitive and there is no GC for wmiinstances over com

Reply:
.Close is required

Reply:
Fixed